dev/io.openliberty.org.testcontainers/bnd.bnd

```diff
@@ -28,8 +28,7 @@ Export-Package: \
 	com.sun.jna.*;version="5.8.0",\
 	org.apache.commons.*;version="1.20",\
 	org.testcontainers.*;version="${testContainerVersion}";-split-package:=merge-last,\
-	org.rnorth.*;version="1.0.8",\
-	org.slf4j.*;version="1.7.30";-split-package:=merge-last
```

Comment:
If we remove this won't we also need to update each FAT suite that depends on TestContainers to include a dependency on the new SLF4J bundle?

Reply:
it isn't the job of the `io.openliberty.org.testcontainers` bundle to provide SLF4J packages. That is the job of the SLF4J bundles.